frontend/components/RoleChooser.tsx

```
@@ -87,37 +93,37 @@ export default function RoleChooser({
-                ? "border-purple-500 bg-purple-900/20" 
-                : "border-gray-600 bg-gray-900/20 hover:border-gray-500"
+                ? "border-purple-500 bg-gradient-to-br from-purple-900/40 to-purple-800/20 shadow-lg shadow-purple-500/20 scale-105" 
+                : "border-gray-600/60 bg-gray-900/30 hover:border-purple-400/60 hover:bg-gray-800/40"
             }`}
             onClick={() => onRoleSelect("professor")}
           >
             <div className="text-center">
-              <div className="mx-auto mb-4 p-4 rounded-full bg-purple-600/20">
-                <svg className={`${iconSize} text-purple-400`} fill="none" stroke="currentColor" viewBox="0 0 24 24">
+              <div className={`mx-auto mb-6 p-5 rounded-2xl bg-gradient-to-br from-purple-600/30 to-purple-500/20 shadow-lg ${selectedRole === "professor" ? "scale-110" : ""} transition-transform duration-300`}>
+                <svg className={`${iconSize} text-purple-300`} fill="none" stroke="currentColor" viewBox="0 0 24 24">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)